ansible/roles/eos/templates/t1-64-lag-tor.j2

```diff
@@ -29,29 +29,6 @@ ipv6 unicast-routing
 !
 ip route vrf MGMT 0.0.0.0/0 {{ mgmt_gw }}
 !
-{% for subnet in range(0, props.tor_subnet_number) %}
-ip route 172.16.{{ tornum }}.{{ subnet }}/32 {{ props.nhipv4 }}
-ipv6 route 20AC:10{{ '%02X' % tornum }}:0:{{ '%02X' % subnet }}::/64 {{ props.nhipv6 }}
-{% endfor %}
-!
-{% if 'vips' in host %}
-{% for subnet in host['vips']['ipv4']['prefixes'] %}
-ip route {{ subnet }} {{ props.nhipv4 }}
-{% endfor %}
-{% set index = 1 %}
-{% for subnet in host['vips']['ipv4']['prefixes'] %}
-ip prefix-list test_vip_{{ index }} seq 1{{ index }} permit {{ subnet }}
-{% set index = index + 1 %}
-{% endfor %}
-{% set index = 1 %}
-{% for subnet in host['vips']['ipv4']['prefixes'] %}
-route-map PREPENDAS permit 2{{ index }}
- match ip address prefix-list test_vip_{{ index }}
- set as-path prepend {{ host['vips']['ipv4']['asn'] }}
-{% set index = index + 1 %}
-{% endfor %}
-{% endif %}
-!
```

Comment:
Can this new `t1-64-lag-tor.j2` configuration support cEOS, like `t1-lag-tor.j2`?

Reply:
I have now updated this file to support cEOS VMs too. I think there isn't a way I can test these changes for cEOS VMs.